### ISSUE TITLE
ai_do_objects_undocked_stuff when objects are already undocked

### ISSUE THREAD

**Issue by DefCynodont** [scp-fs2open/fs2open.github.com#6163]
Regarding this warning, `Call to ai_do_objects_undocked_stuff when objects are already undocked!  Trace out and fix!` 
Calling Jettison Cargo on a ship with nothing docked to it seems to trigger it, furthermore, calling the SEXP AI_Undock on undocked ships should ideally do nothing and have the AI cancel the order.

Could this be fixed in code?

**Comment by Goober5000** [comment]
Yep, will take a look.

**Comment by Goober5000** [comment]
I tried both scenarios using the attached test mission, and both appear to function as expected.  Start the mission and wait for the directives to show up.  Press 1 to jettison and 2 to undock.
[undock_test.fs2.txt](https://github.com/scp-fs2open/fs2open.github.com/files/15410883/undock_test.fs2.txt)

**Comment by Goober5000** [comment]
Any additional information?

**Comment by Goober5000** [comment]
Bumping for a response, especially as we are now in the run-up to 24.2.

**Comment by MjnMixael** [comment]
Kyle, is this still an issue you can repro? Will close this ticket if we don't hear back. Feel free to re-open if it comes up again.

**Comment by DefCynodont** [comment]
yeah, I'll close this. (I forgot about this one too) If it comes up we can re-open.
